lib/jekyll/commands/new.rb

```diff
@@ -57,6 +74,12 @@ def initialized_post_name
 
         private
 
+        def sanitized(args)
```

Comment:
I'm concerned that you're not using things from the Ruby standard library here and are instead using string manipulation. Is there something that prevents you from using `File`, `Dir`, `Pathname`, or `FileUtils`?